### PR TITLE
Add Admin API methods

### PR DESCRIPTION
**Pull request by Nevett**

### CODE REVIEW

File: src/Cronofy/Cronofy.csproj [hunk 1]
```diff
@@ -42,6 +42,9 @@
     <Reference Include="System.Runtime.Serialization" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="CronofyApiKeyClient.cs" />
```
Comment (gshutler) [inline src/Cronofy/Cronofy.csproj]: "API key" could be a number of things. I think this needs to be more explicit, something like `CronofyAdminApiClient` for example.

File: test/Cronofy.Test/CronofyOAuthClientTests/SubmitApplicationVerification.cs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        }
+
+        [Test]
+        public void CanSendCompleteRequest()
+        {
+            this.http.Stub(HttpPost
+                .Url("https://api.cronofy.com/v1/application_verification")
+                .RequestHeader("Authorization", $"Bearer {clientSecret}")
+                .RequestHeader("Content-Type", "application/json; charset=utf-8")
+                .RequestBody(@"{""redirect_uris"":[""https://app.samson.com""],""contact"":{""email"":""brock@samson.com"",""display_name"":""Brock Samson""}}")
```
Comment (gshutler) [inline test/Cronofy.Test/CronofyOAuthClientTests/SubmitApplicationVerification.cs]: Time-travel character references only please 😛

File: test/Cronofy.Test/CronofyOAuthClientTests/SubmitApplicationVerification.cs [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                .ResponseCode(200)
```
Comment (gshutler) [inline test/Cronofy.Test/CronofyOAuthClientTests/SubmitApplicationVerification.cs]: Expected response code is `202`.

File: test/Cronofy.Test/CronofyOAuthClientTests/SubmitApplicationVerification.cs [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+                },
+                RedirectUris = new[]
+                {
+                    "https://app.samson.com"
+                }
+            });
+        }
+
+        [Test]
+        public void CanSendNullDisplayName()
```
Comment (gshutler) [inline test/Cronofy.Test/CronofyOAuthClientTests/SubmitApplicationVerification.cs]: Would say this is more "can omit" or similar as the `null` isn't being sent.

Comment (Nevett) [reply test/Cronofy.Test/CronofyOAuthClientTests/SubmitApplicationVerification.cs]: 👍

File: test/Cronofy.Test/CronofyOAuthClientTests/SubmitApplicationVerification.cs (hunk 3 above)
Comment (Nevett) [reply test/Cronofy.Test/CronofyOAuthClientTests/SubmitApplicationVerification.cs]: 👍

File: src/Cronofy/Cronofy.csproj (hunk 1 above)
Comment (Nevett) [reply src/Cronofy/Cronofy.csproj]: 👍

File: test/Cronofy.Test/CronofyOAuthClientTests/SubmitApplicationVerification.cs (hunk 2 above)
Comment (Nevett) [reply test/Cronofy.Test/CronofyOAuthClientTests/SubmitApplicationVerification.cs]: I like this rule

**Pull request merged**